definition_of_done.md

```diff
@@ -0,0 +1,38 @@
+# Definition of done
+
+This document clarifies, when an product backlog item is done. During scrum development process this gives us a checklist,
+what needs to be done.
+The individual points are a basis for acceptance criteria and are shortened or appended by specific measures as needed.
+
+## Documentation
+* [ ] Public api
```

Comment:
I think not everyone knows how to find the public api.

Maybe we could add a template link like:
[ ] Public api (http://docs.ros.org/melodic/api/[package]/html/)